gapis/api/vulkan/command_buffer_rebuilder.go

```diff
@@ -141,6 +141,14 @@ func allocateNewCmdBufFromExistingOneAndBegin(
 	return newCmdBufID, x, cleanup
 }
 
+func createVkSubpassEndInfo(unused SubpassEndInfoʳ) VkSubpassEndInfo {
+	_ = unused // SubpassEndInfo is currently empty.
```

Comment:
ditto above, this is for the r/w callbacks?

Reply:
same with above.